opendrift/readers/reader_telemac_selafin.py

```diff
@@ -0,0 +1,263 @@
+import numpy as np
+from datetime import datetime, timedelta
+import sys
+from os import path, environ, sep
+from scipy.spatial import cKDTree
+#Activating PYTEL
+dir = environ['HOMETEL']+'{}scripts{}python3'.format(sep,sep)
```

Comment:
Using environment variable (HOMETEL) is not a good solution for all users, can this be solved another way to work out of the box?

Reply:
This variable comes straight from Telemac configuration I don't really know how to improve that. 
Or we have to request to duplicate their code